app/javascript/runtimeBanner/RuntimeBanner.jsx

```diff
@@ -0,0 +1,146 @@
+/* global Runtime */
+
+import { h, Component } from 'preact';
+
+const BANNER_DISMISS_KEY = 'runtimeBannerDismissed';
+
+/**
+ * A banner that will be displayed to provide a deep link into a specified
+ * ConsumerApp based on the Runtime context. If the banner is dismissed it will
+ * keep track of this in localStorage so it won't be rendered again.
+ */
+export class RuntimeBanner extends Component {
```

Comment:
Refactor (blocking): Use a function component instead. We have class components in the codebase but since upgrading Preact last year, we've moved to writing all new components as [function components](https://preactjs.com/guide/v10/components/#functional-components) and [hooks](https://preactjs.com/guide/v10/hooks) (if state is necessary).